docs/src/contributing/releasing.md

```diff
@@ -166,15 +186,37 @@ Edit them until you are happy, then push the tag(s) to GitHub.
  - To show tags: `git tag --list`
  - To push a tag: `git push origin [tag_name]`
 
-### Set Branch `release`
+### If a release-candidate:
 
-By convention, branch `release` should always point to the latest stable commit compatible with the latest release.
+#### Deploy release to testnet machine(s).
+
+(full instructions: TBD)
+
+Announce the release-candidate in a post at talk.neptune.cash.
+
+Ensure that a composer is running using the release-candidate binary.
+
+### If an actual release:
+
+#### Set tag `latest-release`
```

Comment:
With this PR, what used to be the `release` _branch_ is becoming the `latest-release` _tag_. I think a tag makes more sense than a branch. However, the corresponding references to the branch in the README.md should also be updated.

Reply:
yes thx.  and we should probably get buy-off from @aszepieniec regarding the change before merging.  Though to me it is really the only sensible thing... that `latest-release` always points to commit for the latest release.  so hopefully there's consensus on that.